quickstart.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+REVISION=574897620355e2f7486a8cc5586597babd95c85cfa6f148a06470dc80012c2e2
+
+docker run --rm -ti \
+  --name stellar \
```

Comment:
Anyone on new Mac's will need this:
```suggestion
docker run --rm -ti \
  --platform linux/amd64
  --name stellar \
```